mallard/client.py

```diff
@@ -145,6 +146,23 @@ async def on_message(self, message):
         else:
             await self.search(query, message)
 
+    async def on_reaction_add(self, reaction, user):
+        logger.debug("Received reaction event for %d", reaction.message.id)
+
+        if user == self.user:
+            return
+        if reaction.message.author != self.user:
+            return
+        if reaction.emoji == "\U0001F5D1":
```

Comment:
Would advocate making this a constant, also using the `\N{unicode_name}` notation.